### ISSUE TITLE
BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding

### ISSUE THREAD

**Issue by koptimizer** [KGJsGit/my_PaperList#14]
## :clipboard: 논문의 정보를 알려주세요.
- BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding
- Jacob Devlin, Ming-wei Chang, Kenton Lee, and Kristina Toutanova
- arxiv
- 2018-10-11

## :page_with_curl: Abstract(본문)
> We introduce a new language representation model called BERT, which stands for Bidirectional Encoder Representations from
Transformers. Unlike recent language representation models (Peters et al., 2018a; Radford et al., 2018), BERT is designed to pretrain deep bidirectional representations from unlabeled text by jointly conditioning on both left and right context in all layers. As a result, the pre-trained BERT model can be finetuned with just one additional output layer to create state-of-the-art models for a wide range of tasks, such as question answering and language inference, without substantial taskspecific architecture modifications. BERT is conceptually simple and empirically powerful. It obtains new state-of-the-art results on eleven natural language processing tasks, including pushing the GLUE score to 80.5% (7.7% point absolute improvement), MultiNLI accuracy to 86.7% (4.6% absolute improvement), SQuAD v1.1 question answering Test F1 to 93.2 (1.5 point absolute improvement) and SQuAD v2.0 Test F1 to 83.1 (5.1 point absolute improvement).

## :mag_right: 어떤 논문인지 소개해주세요.
- 구글에서 발표한 핫한 NLP 모델로 Transformer 구조를 활용한 Language Representation을 소개합니다.
- 대용량의 Unlabeled Data로 미리 학습 한 후, Labeled Data로 Transfer Learning을 하는 모델입니다.
- BERT를 이용해서 소셜 감성분석에 사용하려고 합니다.

## :key: 핵심 키워드를 적어주세요.
- Transfer Learning, Transformer, Pre-training, Natural Language Process

## :paperclip: URL
- https://arxiv.org/abs/1810.04805
